packages-exp/auth-exp/src/core/strategies/redirect.ts

```diff
@@ -172,3 +172,7 @@ class RedirectAction extends AbstractPopupRedirectOperation {
 
   cleanUp(): void {}
 }
+
+export function _clearOutcomes(): void {
+  redirectOutcomeMap = {};
```

Comment:
Can you call https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/clear instead and keep this is a const?

Reply:
Done